sklearn/ensemble/_bagging.py

```diff
@@ -127,12 +152,25 @@ def _parallel_build_estimators(
             max_samples,
         )
 
-        # Draw samples, using sample weights, and then fit
-        if support_sample_weight:
-            if sample_weight is None:
-                curr_sample_weight = np.ones((n_samples,))
-            else:
-                curr_sample_weight = sample_weight.copy()
+        fit_params_ = fit_params.copy()
+
+        # TODO(SLEP6): remove if condition for unrouted sample_weight when metadata
+        # routing can't be disabled.
+        # 1. If routing is not enabled, we will check if the base
+        # estimator supports sample_weight and use it if it does.
+        # 2. If routing is enabled, we will check if the routing supports sample
+        # weight and use it if it does.
```

Comment:
```suggestion
        # 1. If routing is enabled, we will check if the routing supports sample
        # weight and use it if it does.
        # 2. If routing is not enabled, we will check if the base
        # estimator supports sample_weight and use it if it does.
```

Reply:
I just reordered the numbering in this comment to match the ordering present in the if condition below.